Bigtable/tests/System/BackupTests.php

```diff
@@ -0,0 +1,203 @@
+<?php
+/**
+ * Copyright 2022 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Bigtable\Tests\System;
+
+use Google\Cloud\Bigtable\Admin\V2\Backup;
+use Google\Protobuf\Timestamp;
+use Google\ApiCore\ApiException;
+
+/**
+ * @group bigtable
+ * @group bigatableBackup
```

Comment:
Since all group labels are in smallcase, prefer to keep it this way?
```suggestion
 * @group bigatable-backup
```

Reply:
Yes this seems good, Incorporated.

Thanks